modules/engines/engine-command/src/main/java/org/eclipse/dirigible/engine/command/processor/CommandEngineExecutor.java

```diff
@@ -221,6 +224,11 @@ public String executeCommandLine(String commandLine, Map<String, String> forAddi
 		return result;
 	}
 
+	private boolean shouldLogCommand() {
+		String shouldEnableLogging = Configuration.get("DIRIGIBLE_EXEC_DISABLE_COMMAND_LOGGING");
+		return Boolean.parseBoolean(shouldEnableLogging);
```

Comment:
You could add static import for the DIRIGIBLE_EXEC_COMMAND_LOGGING_ENABLED constant and update the block as follows:
```java
return Boolean.parseBoolean(Configuration.get(DIRIGIBLE_EXEC_COMMAND_LOGGING_ENABLED, "false"));
```

Reply:
I prefer to keep it on two lines as it looks a bit more readable to me and there is no need for the default "false" as parseBoolean handles nulls as well.

As for if it should log or not log, I think it's better to not log by default